x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -323,7 +325,15 @@ export class Embeddable
     if (this.input.onLoad) {
       this.input.onLoad(true);
     }
+    const executionContext = this.deps.executionContext.create({
+      type: this.savedVis.type ?? 'lens',
+      name: this.savedVis.visualizationType ?? '',
```

Comment:
So you want to track it as `name: 'lnsPie'` for example? It's not a unique name.

Reply:
@wylieconlon  I was going to discuss it with you, because `type:lens` is not descriptive enough as lens might have different sub-types. Maybe it makes sense to use `type:lnsPie` or `type:lens_lensPie`?

Reply:
Seems like maybe we could make this parameter name more explicit in a follow-up. Maybe this would be more clear:
- `type` renamed to `feature`
- `name` renamed to `featureSubtype`
- `description` renamed to `instanceDescription`
- `id` renamed to `instanceId`

@mshustov WDYT?